crates/db_schema/src/schema.rs

```diff
@@ -630,6 +630,7 @@ diesel::table! {
         scopes -> Text,
         auto_verify_email -> Bool,
         account_linking_enabled -> Bool,
+        use_pkce -> Bool,
         enabled -> Bool,
```

Comment:
You're missing the needed migrations in `/migrations`. Run `diesel migration generate oauth_pkce` and add the needed SQL. [See here for more info](https://diesel.rs/guides/getting-started).